src/renderer/viz/expressions.js

```diff
@@ -228,7 +230,7 @@ import ViewportFeatures from './expressions/viewportFeatures';
 import XYZ from './expressions/xyz';
 
 import Zoom from './expressions/zoom';
-import Sprite from './expressions/sprite';
+import {Sprite, SVG} from './expressions/sprite';
```

Comment:
There is a missing space between curly brackets. 

Also, I think we should avoid having multiple classes in the same file as a convention. I know it is a tiny class, and that we have other files with many classes, but it is a continuous improvement.

Reply:
No, please, nooo, not Java :sob: :sob: :sob:

Reply:
Is not Java, is beautiful code structure ❤️

Reply:
It's like Vogon poetry to my eyes :slightly_smiling_face: